metecho/api/jobs.py

```diff
@@ -817,7 +823,7 @@ def refresh_commits(*, project, branch_name, originating_user_id):
     This should only run when we're notified of a force-commit. It's the
     nuclear option.
     """
-    from .models import Epic, Task
+    from .models import Epic
```

Comment:
Why is this one imported in the job, while the others are imported at the file-level?

Reply:
Sometimes importing at the function level is necessary to avoid import loops; for example if `models.py` imports from `jobs.py` but `jobs.py` also imports a model. In this case I think we can safely move it to the top level

Reply:
Yeah, it looks like all imports of `jobs.py` in `models.py` are function level, so I'll update this.